homeassistant/components/tuya/sensor.py

```diff
@@ -1129,6 +1155,8 @@ def __init__(
             self._type = DPType.INTEGER
             if description.native_unit_of_measurement is None:
                 self._attr_native_unit_of_measurement = int_type.unit
+            if description.scale is not None:
+                self._type_data.scale = description.scale
```

Comment:
This seems odd / incorrect / hacky.

The scale is part of the data type exposed by the API, why is this overriding the provided data?

../Frenck

Reply:
Aah it was in the additional description.

> The tuya api delivers the scale field of the temperature sensor with a wrong value of 1. 

This is an issue with the device or upstream Tuya API. Please contact Tuya support that from your device about this as this is a bug.

This is not something we just patch on our end, as it can have side effects for other devices.

../Frenck

Reply:
@frenck I have decided to remove the temperature sensor from this PR.
It's not the most relevant sensor of the micro inverter and I'm not sure how easy tuya or the manufacturer of this device might be able to change this in all relevant places.